selfservice/strategy/oidc/strategy_login.go

```diff
@@ -109,10 +109,43 @@ func (s *Strategy) processLogin(w http.ResponseWriter, r *http.Request, a *login
 		return nil, s.handleError(w, r, a, provider.Config().ID, nil, errors.WithStack(herodot.ErrInternalServerError.WithReason("The password credentials could not be decoded properly").WithDebug(err.Error())))
 	}
 
+
+	// UPDATE TOKEN
+	var it string
+	if idToken, ok := token.Extra("id_token").(string); ok {
```

Comment:
Could you please add a test for this? :)

Reply:
yes sure